cli/gencert/gencert.go

```diff
@@ -31,6 +31,8 @@ Usage of gencert:
 
 Arguments:
         CSRJSON:    JSON file containing the request, use '-' for reading JSON from stdin
+	cert:	    file or environment variable containing the CA certificate -- use '[file:]fname' or 'env:varname'
```

Comment:
update the helper text here https://github.com/cloudflare/cfssl/blob/master/cli/config.go#L78 (and line 79 too)
and then we don't need to specify this at `gencert` because it will be available to all cil command that uses `ca` and `ca-key`